apps/cowswap-frontend/src/pages/LimitOrders/AlternativeLimitOrder.tsx

```diff
@@ -1,18 +1,41 @@
 import { useCallback } from 'react'
 
+import { UI } from '@cowprotocol/ui'
+
+import styled from 'styled-components/macro'
+
 import { LimitOrdersWidget } from 'modules/limitOrders'
 import { useHideAlternativeOrderModal } from 'modules/trade/state/alternativeOrder'
 
 import { NewModal } from 'common/pure/NewModal'
 
+const MODAL_MAX_WIDTH = 450
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-flow: column wrap;
+  width: 100%;
+  max-width: ${MODAL_MAX_WIDTH}px;
+`
+
 export function AlternativeLimitOrder() {
   const hideAlternativeOrderModal = useHideAlternativeOrderModal()
 
   const onDismiss = useCallback(() => hideAlternativeOrderModal(), [hideAlternativeOrderModal])
 
   return (
-    <NewModal title={'Recreate order'} onDismiss={onDismiss} modalMode>
-      <LimitOrdersWidget />
-    </NewModal>
+    <Wrapper>
+      <NewModal
+        title={'Recreate order'}
+        onDismiss={onDismiss}
+        modalMode
+        maxWidth={MODAL_MAX_WIDTH}
+        backgroundColor={UI.COLOR_PAPER_DARKER}
+        border={`1px solid var(${UI.COLOR_PAPER})`}
+        contentPadding={'10px'}
```

Comment:
We don't usually do styling as props.
Could this be a new component with that abstracted away in a "wrapper"?
If it's overkill or not a good idea, I'll abide by your decision.